scss/modules/_if-focus-interface.scss

```diff
@@ -0,0 +1,4 @@
+.if-focus-interface {
```

Comment:
This is future code, I've removed it for now. 

I have updated this in the following feature: feature/improve-code-and-functionality